be/src/exec/aggregator.cpp

```diff
@@ -413,6 +414,23 @@ Status Aggregator::prepare(RuntimeState* state, ObjectPool* pool, RuntimeProfile
             Expr* expr = nullptr;
             ExprContext* ctx = nullptr;
             RETURN_IF_ERROR(Expr::create_tree_from_thrift(_pool, desc.nodes, nullptr, &node_idx, &expr, &ctx, state));
+
+            if (state->is_jit_enabled()) {
+                auto* jit_engine = JITEngine::get_instance();
```

Comment:
JITEngine::get_instance() would return nullptr?

Reply:
No, the jit_engine is a global singleton.